### PR TITLE
Fixes #165 bug in test script

### LINKED ISSUE #165: Mocha fails when testing crypter.js function
### This is a
- [x] Bug report
- [ ] Feature request

### Description

image

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue #165 

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix 
- [ ] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: util/crypter.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
-const crypto = require('crypto');
+var dotenv = require('dotenv');
+dotenv.config({ path: './.env'});
```
Comment (sch00lb0y) [inline util/crypter.js]: export the environment in test.js instead of importing environmental variable in crypter.js will look good.

**Comment by imujjwal96** [comment]
@sch00lb0y  can you please elaborate?

**Comment by sch00lb0y** [comment]
@imujjwal96
 ```js
var dotenv = require('dotenv');
dotenv.config({ path: './.env'});
``` 
hereafter if we write any test case which depend on environment varible we have to duplicate this block in that module as well so, if you move this block to test.js there won't be any duplication and also  code will look clean

**Comment by imujjwal96** [comment]
What if i want to run the crypt functions in the front end? How will I access the environment variable then?

**Comment by sch00lb0y** [comment]
we reused validation.js  in both front-end and backend because it is just a plain javascript which is not dependent on any other node js module but here it using `crypto` module so we can't use in frontend and also environmental variable can't be exported to front-end.

**Comment by imujjwal96** [comment]
I tried what you mentioned, and it is not working.
image

**Comment by sch00lb0y** [comment]
@imujjwal96 try by moving the block to the top to the file. I mean before crypter.js import
```js
var dotenv = require("dotenv");
dotenv.config({path: './.env'})
```

**Comment by imujjwal96** [comment]
@sch00lb0y I've updated it. Thanks for the suggestion. :+1:

**Pull request merged**